src/main/scala/mesosphere/marathon/MarathonSchedulerActor.scala

```diff
@@ -306,23 +310,23 @@ class MarathonSchedulerActor private (
   }
 
   def deploy(driver: SchedulerDriver, plan: DeploymentPlan): Unit = {
-    deploymentRepository.store(plan).foreach { _ =>
+    deploymentRepository.store(plan).foreach { done =>
```

Comment:
It could, a ton of Unit could be done. I accidentally named the useless 'done' after running into some strange errors and forgot to remove it.